hello-world/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "@tsconfig/node14/tsconfig.json",
+  "version": "4.2.2",
+  "compilerOptions": {
+    "emitDecoratorMetadata": false,
+    "experimentalDecorators": false,
+    "declaration": true,
+    "declarationMap": true,
+    "sourceMap": true,
+    "composite": true,
+    "incremental": true,
+    "rootDir": "./src",
+    "outDir": "./lib"
```

Comment:
You'll need to add the `@activities` alias here.
We don't really need our project to be composite.
You should probably exclude `node_modules`.